larray/inout/array.py

```diff
@@ -480,6 +481,11 @@ def read_hdf(filepath_or_buffer, key, fill_value=np.nan, na=np.nan, sort_rows=Fa
     sort_columns : bool, optional
         Whether or not to sort the columns alphabetically (sorting is more efficient than not sorting).
         Defaults to False.
+    type : { 'LArray' | 'Axis' | 'Group' }, optional
+        Type of object to be read. Defaults to 'LArray'.
```

Comment:
It is nice to have this option, but this shouldn't be required for HDF. We should store the type in the HDF as extra metadata.

Reply:
Reminder for myself : http://pandas-docs.github.io/pandas-docs-travis/cookbook.html#hdfstore